fhirstore/search/search_methods.py

```diff
@@ -33,6 +33,8 @@ def build_element_query(key, value):
     numeric_prefix = re.search(r"^(gt|lt|ge|le)([0-9].*)$", f"{value}")
     eq_prefix = re.search(r"^(eq)([0-9].*)$", f"{value}")
     special_prefix = re.search(r"^(ne|sa|eb|ap)([0-9].*)$", f"{value}")
+    pipe_suffix = re.search(r"(.*)\|(.*)", f"{value}")
```

Comment:
why do you use templated strings here ? isn't this the same as : `pipe_suffix = re.search(r"(.*)\|(.*)", value)` ? or are you using this to convert value to a string in case it isn't ?

Reply:
Yes, for instance if the value is a quantity, it won't be a string. But it is not necessary with the pipe.